prebid-server/developers/add-new-analytics-module.md

```diff
@@ -10,23 +10,23 @@ This document describes how to add a new Analytics module to Prebid Server.
 
 ### 1. Define config params
 
-Analytics modules are enabled through the [Configuration](./configuration.md).
-You'll need to define any properties in [config/config.go](../../config/config.go)
+Analytics modules are enabled through the [Configuration](/prebid-server/developers/configuration.html).
```

Comment:
configuration.md wasn't supposed to come over. Can we remove that file and point the links to it back to the PBS github?